lib/badge/badge.test.tsx

```diff
@@ -40,6 +40,11 @@ describe('<Badge />', () => {
 
     wrapper.setProps({ style: { zIndex: '1' } });
```

Comment:
s there a point to test zIndex here again, or is it just duplication?

Reply:
I'm testing 0, 1 and 2 CSS properties across all the assertions—just to be thorough.